specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/usages.json

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
This new file is not referenced from a readme.md file. https://dev.azure.com/azure-sdk/public/_build/results?buildId=51672&view=logs

Reply:
updated to have readme.md reference usages.json